src/Payum/Paypal/ExpressCheckout/Nvp/Api.php

```diff
@@ -529,6 +534,26 @@ public function doCapture(array $fields)
     }
 
     /**
+     * Require: TRANSACTIONID
+     *
+     * @param array $fields
+     *
+     * @return array
+     */
+    public function doVoid(array $fields)
+    {
+        $fields['METHOD']  = 'DoVoid';
+        if (empty($fields['AUTHORIZATIONID'])) {
+            $fields['AUTHORIZATIONID'] = $fields['TRANSACTIONID'];
```

Comment:
Done.  I also noticed the action was missing tests, so added those.  I think I covered all the cases.